decoders/connector/lansitec/asset-management-tracker-5400mah/v1.0.0/payload.js

```diff
@@ -1,16 +1,16 @@
 function float16(bytes) {

```

Comment:
This decoder must have a unit test, here is an example of a unit test you can use.
```ts
import { describe, test, expect } from "vitest";

import { decoderRun } from "../../../../../src/functions/decoder-run";

const file_path = "decoders/connector/your-path/payload.js" as const;

describe("Shall not be parsed", () => {
  let payload = [{ variable: "shallnotpass", value: "04096113950292" }];
  payload = decoderRun(file_path, { payload });
  test("Output Result", () => {
    expect(Array.isArray(payload)).toBe(true);
  });

  test("Not parsed Result", () => {
    expect(payload).toEqual([{ variable: "shallnotpass", value: "04096113950292" }]);
  });
});
```

If you can add a unit test for the payload parser, that would be great.